redundant-bmc/src/manager.cpp

```diff
@@ -42,4 +48,15 @@ sdbusplus::async::task<> Manager::doHeartBeat()
     co_return;
 }
 
+Manager::Role Manager::determineRole()
+{
+    role_determination::Input input{.bmcPosition = services->getBMCPosition()};
```

Comment:
There is chance to get exception, do you want to handle that or crash application?

Reply:
I'll look into it.

Reply:
If I catch an exception, I guess about all I can do is become Passive since I won't have enough information to do anything else.  That's probably better that letting it die so there is no Role on dbus/redfish at all. Will fix.